style.css

```diff
@@ -868,15 +868,22 @@ a:visited .nav-title,
 .comment-metadata a:hover,
 .comment-metadata a.comment-edit-link:focus,
 .comment-metadata a.comment-edit-link:hover,
-.comment-reply-link:focus,
-.comment-reply-link:hover,
 .widget_authors a:focus strong,
 .widget_authors a:hover strong,
 .project-terms a:focus,
 .project-terms a:hover {
 	border-bottom-color: transparent;
 	background-color: #767676;
 	color: #fff;
+	transition: all 0.3s linear;
+}
+
+.comment-reply-link:hover{
```

Comment:
Expected a single space before the open `{` brace

i.e. `.comment-reply-link:hover {`, not `.comment-reply-link:hover{`